jupyter-js-widgets/src/utils.js

```diff
@@ -143,42 +143,22 @@ function reject(message, log) {
 }
 
 /**
- * Apply MathJax rendering to an element, and optionally set its text
+ * Apply MathJax rendering to an element, and optionally set its text.
  *
  * If MathJax is not available, make no changes.
  *
- * Returns the output any number of typeset elements as an array or undefined if
- * MathJax was not available.
- *
  * Parameters
  * ----------
- * element: Node, NodeList, or jQuery selection
- * text: option string
+ * element: Node
+ * text: optional string
  */
 function typeset(element, text) {
-    if (arguments.length > 1) {
-        if (element.length) {
-            for (var i = 0; i < element.length; ++i) {
-                var el = element[i];
-                el.textContent = text;
-            }
-        } else {
-            element.textContent = text;
-        }
+    if (text) {
```

Comment:
Let's unconditionally set the text. Suppose I have a label widget which sets the content based on a text string coming from python. If the user for whatever reason sets the content to `''`, this function will now keep the original text since `''` is false in Javascript.